content/en/docs/tasks/policy-enforcement/rate-limit/index.md

```diff
@@ -269,7 +269,7 @@ spec:
           vhost:
             name: "inbound|http|9080"
              route:
-              action: ANY
+               action: ANY
```

Comment:
I don't think this change is correct. Did you see https://github.com/istio/istio.io/pull/9044 ?

Reply:
Yes, #9044 is correct.